### ISSUE TITLE
The size of public tarballs has increased a lot, up to almost 1GB!

### ISSUE THREAD

**Issue by lrineau** [CGAL/cgal#682]
```shell
cgal /srv/CGAL/www/Members/Releases $ ls CGAL-4.[678]*ublic/*xz -lSr | tail -15
-rw-r--r--. 1 lrineau geometryfactory  13694500 Oct 18 21:01 CGAL-4.8-I-24-public/CGAL-4.8-dev.tar.xz
-rw-r--r--. 1 lrineau geometryfactory  13719992 Sep 11 21:01 CGAL-4.7-Ic-142-public/CGAL-4.7-beta2.tar.xz
-rw-r--r--. 1 lrineau geometryfactory  13720356 Sep 16 21:01 CGAL-4.7-Ic-146-public/CGAL-4.7-beta2.tar.xz
-rw-r--r--. 1 lrineau geometryfactory 212428444 Nov  8 21:07 CGAL-4.8-I-42-public/CGAL-4.8-dev.tar.xz
-rw-r--r--. 1 lrineau geometryfactory 275208124 Nov 15 21:07 CGAL-4.8-I-48-public/CGAL-4.8-dev.tar.xz
-rw-r--r--. 1 lrineau geometryfactory 343084092 Nov 22 21:09 CGAL-4.8-I-54-public/CGAL-4.8-dev.tar.xz
-rw-r--r--. 1 lrineau geometryfactory 411212452 Nov 29 21:10 CGAL-4.8-I-60-public/CGAL-4.8-dev.tar.xz
-rw-r--r--. 1 lrineau geometryfactory 479303732 Dec  6 21:12 CGAL-4.8-I-66-public/CGAL-4.8-dev.tar.xz
-rw-r--r--. 1 lrineau geometryfactory 547382704 Dec 13 21:14 CGAL-4.8-I-72-public/CGAL-4.8-dev.tar.xz
-rw-r--r--. 1 lrineau geometryfactory 617166988 Dec 20 21:18 CGAL-4.8-I-78-public/CGAL-4.8-dev.tar.xz
-rw-r--r--. 1 lrineau geometryfactory 686969088 Dec 27 21:17 CGAL-4.8-I-84-public/CGAL-4.8-dev.tar.xz
-rw-r--r--. 1 lrineau geometryfactory 756748712 Jan  3 21:19 CGAL-4.8-I-90-public/CGAL-4.8-dev.tar.xz
-rw-r--r--. 1 lrineau geometryfactory 827657400 Jan 10 21:24 CGAL-4.8-I-96-public/CGAL-4.8-dev.tar.xz
-rw-r--r--. 1 lrineau geometryfactory 898567580 Jan 17 21:22 CGAL-4.8-I-102-public/CGAL-4.8-dev.tar.xz
-rw-r--r--. 1 lrineau geometryfactory 969505544 Jan 24 21:24 CGAL-4.8-I-108-public/CGAL-4.8-dev.tar.xz
```

**Comment by lrineau** [comment]
Cc @bo0ts

**Comment by lrineau** [comment]
If you do have a shell access to the machine, but you are a CGAL developer,  you can see the tarballs here: https://cgal.geometryfactory.com/CGAL/Members/Releases/

**Comment by bo0ts** [comment]
This issue seems to be present since the 25th of October. The first release affected is I-30.

It seems that 30d4951f3f72692fb30c89e6289747bd69640e15 introduced the problem. The old code moved the directory `${public_release_name}` to `${public_release_name}-NSIS` and removed it afterwards. The new code just uses `${public_release_name}` without renaming it, hence the removal at the end fails.

Also, we really should require all bash scripts to have `set -e`.

**Comment by lrineau** [comment]
I will remove all those files, now that we know what is the bug:
```
cgal /srv/CGAL/www/Members/Releases $ ls -lrt CGAL-4.[678]*ublic/*xz(LM+200)
-rw-r--r--. 1 lrineau geometryfactory  212428444 Nov  8 21:07 CGAL-4.8-I-42-public/CGAL-4.8-dev.tar.xz
-rw-r--r--. 1 lrineau geometryfactory  275208124 Nov 15 21:07 CGAL-4.8-I-48-public/CGAL-4.8-dev.tar.xz
-rw-r--r--. 1 lrineau geometryfactory  343084092 Nov 22 21:09 CGAL-4.8-I-54-public/CGAL-4.8-dev.tar.xz
-rw-r--r--. 1 lrineau geometryfactory  411212452 Nov 29 21:10 CGAL-4.8-I-60-public/CGAL-4.8-dev.tar.xz
-rw-r--r--. 1 lrineau geometryfactory  479303732 Dec  6 21:12 CGAL-4.8-I-66-public/CGAL-4.8-dev.tar.xz
-rw-r--r--. 1 lrineau geometryfactory  547382704 Dec 13 21:14 CGAL-4.8-I-72-public/CGAL-4.8-dev.tar.xz
-rw-r--r--. 1 lrineau geometryfactory  617166988 Dec 20 21:18 CGAL-4.8-I-78-public/CGAL-4.8-dev.tar.xz
-rw-r--r--. 1 lrineau geometryfactory  686969088 Dec 27 21:17 CGAL-4.8-I-84-public/CGAL-4.8-dev.tar.xz
-rw-r--r--. 1 lrineau geometryfactory  756748712 Jan  3 21:19 CGAL-4.8-I-90-public/CGAL-4.8-dev.tar.xz
-rw-r--r--. 1 lrineau geometryfactory  827657400 Jan 10 21:24 CGAL-4.8-I-96-public/CGAL-4.8-dev.tar.xz
-rw-r--r--. 1 lrineau geometryfactory  898567580 Jan 17 21:22 CGAL-4.8-I-102-public/CGAL-4.8-dev.tar.xz
-rw-r--r--. 1 lrineau geometryfactory  969505544 Jan 24 21:24 CGAL-4.8-I-108-public/CGAL-4.8-dev.tar.xz
-rw-r--r--. 1 lrineau geometryfactory 1040403864 Jan 31 21:31 CGAL-4.8-I-114-public/CGAL-4.8-dev.tar.xz
```

**Comment by sloriot** [comment]
It should be fixed by 3ce3687
The temporary directory was manually removed.
Let's wait for CGAL-4.8-I-120 before closing the issue.

**Comment by sloriot** [comment]
The public release was successfully created but the directory `/home/lrineau/CGAL/create_internal_release/CGAL-4.8-dev` was not removed by the run of Sunday. The windows installer was not generated. I think the pb is related to an error while trying to run docker to generate the windows installer (because the tgz was also not removed) which made the script stop and the removals were not done.
